src/docfx/docset/Docset.cs

```diff
@@ -67,6 +67,11 @@ internal class Docset
         /// </summary>
         public IReadOnlyDictionary<string, string> ReversedRoutes { get; }
 
+        /// <summary>
+        /// Gets the moniker range parser
+        /// </summary>
+        public MonikerRangeParser MonikerRangeParser => _monikerRangeParser.Value;
```

Comment:
>MonikerRangeParser [](start = 34, length = 18)

why need expose Parser if we already get Provider?

Reply:
parser will be used to parse monikerRange in `zone` syntax

---
In reply to: [232539893](https://github.com/dotnet/docfx/pull/3596#discussion_r232539893) [](ancestors = 232539893)

Reply:
can we also integrate this part to Provider?

---
In reply to: [232541932](https://github.com/dotnet/docfx/pull/3596#discussion_r232541932) [](ancestors = 232541932,232539893)